### PR TITLE
Create Mad Skill Chinese translation Key file

### PR DESCRIPTION
**Pull request by subtank**
Create MadSkills Chinese translation because I was mad with half Chinese half English in game.

### CODE REVIEW

**Comment by subtank** [comment]
@duduluu I just create a Chinese translation for this mod.Can you check it again?

File: Languages/ChineseSimplified/Keyed/MadSkills_Keyed_CN.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+	<MadSkills_SettingsCategory>Mad Skills</MadSkills_SettingsCategory>
+	<MadSkills_DailyXPSaturationThresholdLabel>每日经验值上限</MadSkills_DailyXPSaturationThresholdLabel>
+	<MadSkills_DailyXPSaturationThresholdTip>边缘世界默认值：4000
+当你达到这里的设定值后，直到午夜前都会按照下面一个设置的比例获取经验。</MadSkills_DailyXPSaturationThresholdTip>
+	<MadSkills_SaturatedXPMultiplierLabel>防沉迷</MadSkills_SaturatedXPMultiplierLabel>
+  <!--Chinese pun for this translation-->
+	<MadSkills_SaturatedXPMultiplierTip>边缘世界默认值：20%
+超过每日经验上限部分的经验获取比例。设定为100%则是移除上限。</MadSkills_SaturatedXPMultiplierTip>
+	<MadSkills_MultiplierLabel>经验衰减速率</MadSkills_MultiplierLabel>
```
Comment (duduluu) [inline Languages/ChineseSimplified/Keyed/MadSkills_Keyed_CN.xml]: 速率 => 系数 or 倍率

File: Languages/ChineseSimplified/Keyed/MadSkills_Keyed_CN.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	<MadSkills_MultiplierTip>边缘世界默认值：100%
+0是不衰减50%是衰减速率减半，你也可以疯狂到设定到高倍数来玩心跳。</MadSkills_MultiplierTip>
```
Comment (duduluu) [inline Languages/ChineseSimplified/Keyed/MadSkills_Keyed_CN.xml]: 0是不衰减，50%是衰减速率减半，200%表述双倍衰减，你也可以疯狂到设定到高倍数来玩心跳。

File: Languages/ChineseSimplified/Keyed/MadSkills_Keyed_CN.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	<MadSkills_TieredLabel>降级设定</MadSkills_TieredLabel>
+	<MadSkills_TieredTip>边缘世界默认值：关
+防止降级</MadSkills_TieredTip>
+	<MadSkills_AlternativeGreatMemoryDescription>{PAWN_nameDef} 有一个更好的记性。 {PAWN_pronoun} 会比别人学得快。</MadSkills_AlternativeGreatMemoryDescription>
```
Comment (duduluu) [inline Languages/ChineseSimplified/Keyed/MadSkills_Keyed_CN.xml]: {PAWN_nameDef}对细节有极好的记忆力。{PAWN_pronoun}学起新技能来会比其他人稍微快一点点。

File: Languages/ChineseSimplified/Keyed/MadSkills_Keyed_CN.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	<MadSkills_AlterGreatMemoryLabel>替代特性“过目不忘”</MadSkills_AlterGreatMemoryLabel>
+	<MadSkills_AlterGreatMemoryTip>边缘世界默认值：off
+将旧版“过目不忘”的减半经验衰减速率变为增加25%学习速度。</MadSkills_AlterGreatMemoryTip>
```
Comment (duduluu) [inline Languages/ChineseSimplified/Keyed/MadSkills_Keyed_CN.xml]: 修改“过目不忘“的表现：将“经验衰减速率减半”替换为“学习速度增加25%”。

File: Languages/ChineseSimplified/Keyed/MadSkills_Keyed_CN.xml (hunk 1 above)
Comment (subtank) [reply Languages/ChineseSimplified/Keyed/MadSkills_Keyed_CN.xml]: 忘了怎么改PR了。。。duduluu你来改吧。

**Comment by duduluu** [comment]
Hi, @Ratysz 
It's OK. Merge please!

**Comment by Ratysz** [comment]
Sorry, no. I really appreciate your work, but I don't want to include external translations into my mods anymore - I can't make changes to them if I change something in the mod, and neither waiting for translators nor having releases with broken translations are things I want to do; not to mention having the translators wait to see their translation inplemented until the next time I decide to update.

I am fluent in English and Russian, so those are the only languages I'll include with my mods themselves. If you want to add another translation, please make a translation mod instead. Thank you.

**Comment by subtank** [comment]
@duduluu 你用这个来搞一个中文化mod吧。懒得编译啥的了。

**Pull request closed without merging**